cmd/generic-handlers.go

```diff
@@ -624,6 +626,11 @@ func (f bucketForwardingHandler) ServeHTTP(w http.ResponseWriter, r *http.Reques
 		f.handler.ServeHTTP(w, r)
 		return
 	}
+	// CopyObject request
```

Comment:
A bit more elaborate comment is better.. why we need to handle CopyObject as a special case.